packages/vscode-extension/src/project/project.ts

```diff
@@ -372,6 +376,52 @@ export class Project implements Disposable, MetroDelegate, ProjectInterface {
     this.deviceSession?.onActiveFileChange(filename, followEnabled);
   }
 
+  public async getCommandsCurrentKeyBinding(commandName: string) {
+    const packageJsonPath = path.join(extensionContext.extensionPath, "package.json");
+    const extensionPackageJson = require(packageJsonPath);
```

Comment:
note that require will load the file only once and then subsequent calls will just load the cached version.

Reply:
that's okay since keybindings set in package.json are static.